frontend/src/Components/Login/Login.jsx

```diff
@@ -16,9 +16,13 @@ const Login = ({ setAccessToken }) => {
    const [loading, setLoading] = useState(false);
    const [successfulLogin, setSuccessfulLogin] = useState("");
```

Comment:
Makes more sense to be boolean? Or successfulLoginMsg just for better readability.

Reply:
Changed name to successfullMsg, it does sound and look better, thanks a lot.